lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryImpl.java

```diff
@@ -158,6 +158,7 @@
   private boolean addAttachmentColumnsToSelectClauseForOrdering = false;
   private boolean addClassificationNameToSelectClauseForOrdering = false;
   private boolean addAttachmentClassificationNameToSelectClauseForOrdering = false;
+  private boolean addWorkbasketNameForOrdering = false;
```

Comment:
should we name this "addWorkbasketNameToSelectClauseForOrdering" to be consistent with the rest?

Aren't we missing a "joinWithWorkbaskets" property? (Which should probably also be handled in the setupJoinAndOrderParameters() method)

Reply:
The way I see it, a different handling of "joinWithWorkbasket" and "addWorkbasketNameToSelectClauseForOrdering" is not necessary. When joining the workbaskets in the queries we always want to also select the workbasket name. 
The  reason is that we don't implement the option to search for a certain workbasket name in which case we would join but not select. We only want to be able to order the results by the workbasket names (join + select).

Reply:
"addWorkbasketNameToSelectClauseForOrdering" would be consistent, in my view "addWorkbasketNameForOrdering" is easier and could emphasize the difference to attachments and classifications...